babel/src/test/java/org/apache/calcite/test/BabelParserTest.java

```diff
@@ -342,4 +342,9 @@ private void checkParseInfixCast(String sqlType) {
     sql(sql).ok(expected);
   }
 
+  @Test public void testInlineModSyntax() {
```

Comment:
nit: Suggest we add a test for both floating point and integer-only arguments to MOD.